app/Models/Icon.php

```diff
@@ -34,7 +34,9 @@ public function set(): BelongsTo
 
     public static function relatedIcons(self $icon): Collection
     {
-        return static::search($icon->keywords)->get();
+        return static::search($icon->keywords)
+            ->get()
+            ->where('id', '!=', $icon->id);
```

Comment:
Shouldn't we do the where before the `get` so it's excluded before the search is done?

Reply:
I couldn't find a way to tell scout to exclude something.

There is only a `where('key', 'value)`

Reply:
I guess it doesn't really matter for only the one icon.